src/internal/cluster/data.go

```diff
@@ -64,48 +72,50 @@ iterator:
 		// Inject into all the pods
 		for _, pod := range pods {
 			// Try to use the embedded kubectl if we can
-			binPath, err := utils.GetFinalExecutablePath()
+			zarfBinPath, err := utils.GetFinalExecutablePath()
+			kubectlBinPath := "kubectl"
 			if err != nil {
-				message.Warnf("Unable to get the final executable path, falling back to host kubectl: %s", err)
-				binPath = ""
+				message.Warnf("Unable to get the zarf executable path, falling back to host kubectl: %s", err)
 			} else {
-				binPath = fmt.Sprintf("%s tools ", binPath)
+				kubectlBinPath = fmt.Sprintf("%s tools kubectl", zarfBinPath)
 			}
 
-			kubectlExec := fmt.Sprintf("%skubectl exec -i -n %s %s -c %s ", binPath, data.Target.Namespace, pod, data.Target.Container)
-			tarExec := fmt.Sprintf("tar c%s", tarCompressFlag)
-			untarExec := fmt.Sprintf("tar x%svf - -C %s", tarCompressFlag, data.Target.Path)
+			kubectlCmd := fmt.Sprintf("%s exec -i -n %s %s -c %s ", kubectlBinPath, data.Target.Namespace, pod, data.Target.Container)
+
+			// Note that each command flag is separated to provide the widest cross-platform tar support
+			tarCmd := fmt.Sprintf("tar -c%s -f -", tarCompressFlag)
+			untarCmd := fmt.Sprintf("tar -x%s -v -f - -C %s", tarCompressFlag, data.Target.Path)
```

Comment:
I know this works because we have a space in the flag (that gets set above earlier in the function), but since we have the power to string format this, I prefer something like `-c %s`  over `-c%s` since that more clearly shows that it's two different flags instead of trying to customize what the name of a single flag is.